CRM/Contact/BAO/Contact.php

```diff
@@ -2762,15 +2758,14 @@ public static function getCountComponent($component, $contactId, $tableName = NU
         return CRM_Mailing_BAO_Mailing::getContactMailingsCount($params);
 
       default:
-        $custom = explode('_', $component);
-        if ($custom['0'] = 'custom') {
```

Comment:
Hmm, that conditional wasn't doing what it appeared to be doing. Fixed.

Reply:
@colemanw OK - this is doing my head in now - universe tells me that lineitemeditor calls this function & passes in `$tableName` - oh - my brain hurts...

 `    if ($custom['0'] = 'custom') {` is the equivalent of there not being an if - so it never should have worked but it did because of that typo - argh

Reply:
Ok, in that case I've removed the always-true conditional rather than fixing it, and left a todo in the code about lineItemEditor